src/targets/gpu/compile_ops.cpp

```diff
@@ -185,7 +203,10 @@ void par_compile(std::size_t n, F f)
 {
     if(n == 0)
         return;
-    par_for(n, n / value_of(MIGRAPHX_GPU_COMPILE_PARALLEL{}, n), f);
+    auto d = value_of(MIGRAPHX_GPU_COMPILE_PARALLEL{});
```

Comment:
```suggestion
    auto d = value_of(MIGRAPHX_GPU_COMPILE_PARALLEL{}, n);
```

Reply:
That wont work, because the `n` will be cached when `MIGRAPHX_GPU_COMPILE_PARALLEL` is not set.